frontend/src/components/Home/ContinuePlaying.vue

```diff
@@ -14,7 +14,7 @@ const { continuePlayingRoms } = storeToRefs(romsStore);
 const gridContinuePlayingRoms = isNull(
   localStorage.getItem("settings.gridContinuePlayingRoms"),
 )
-  ? true
+  ? false
   : localStorage.getItem("settings.gridContinuePlayingRoms") === "true";
```

Comment:
With this new logic everywhere, is this now equivalent to...?

```ts
const gridContinuePlayingRoms = localStorage.getItem("settings.gridContinuePlayingRoms") === "true";
```

Reply:
yeah i think so, though there's no hard to keeping it like this for code consistency